ge05_team01/models/calculate_distance.py

```diff
@@ -0,0 +1,30 @@
+from odoo import fields, models, api
+
+san_francisco = ["California", "Oregon", "Washington",
+                 "Nevada", "Arizona", "Utah", "Idaho", "Montana", "Wyoming", "Colorado", "New Mexico", "Sonora", "Sinaloa", "Nayarit", "Baja California", "Baja California Sur"]
+buffalo = ["New York", "Pennsylvania", "Ohio", "Michigan", "Indiana", "West Virginia", "Vermont",
```

Comment:
Using lists definitely works, however, can you come up with a solution using the odoo fields? That way we can store them in the DB and in the future, maybe add to the lists without messing with code.

Reply:
This is something that i'll try to implement in my free time hahaha by the moment we are going to start with task 7, what do you think master? x)